proj/xc12/BoE.xcodeproj/project.pbxproj

```diff
@@ -325,6 +325,7 @@
 		91FCC8F418FF0866007026CE /* pc.appleevents.mm in Sources */ = {isa = PBXBuildFile; fileRef = 91FCC8F318FF069A007026CE /* pc.appleevents.mm */; };
 		91FD417423DFC9D4003D5B8A /* boe.ui.cpp in Sources */ = {isa = PBXBuildFile; fileRef = 91FD417223DFC5C1003D5B8A /* boe.ui.cpp */; };
 		91FE0E3823F084B70084CA6B /* drawable_manager.cpp in Sources */ = {isa = PBXBuildFile; fileRef = 91FE0E3523F084B70084CA6B /* drawable_manager.cpp */; };
+		D384F6A32C1B9D4000A806C3 /* replay.cpp in Sources */ = {isa = PBXBuildFile; fileRef = D384F6A22C1B9D4000A806C3 /* replay.cpp */; };
```

Comment:
Did you do this in Xcode, with a tool, or manually?

Reply:
In Xcode.